repository/test-src/org/pentaho/platform/repository2/unified/DefaultUnifiedRepositoryBase.java

```diff
@@ -115,7 +111,7 @@
 @ContextConfiguration( locations = { "classpath:/repository.spring.xml",
     "classpath:/repository-test-override.spring.xml" } )
 @SuppressWarnings( "nls" )
-public class DefaultUnifiedRepositoryBase implements ApplicationContextAware {
+public abstract class DefaultUnifiedRepositoryBase implements ApplicationContextAware {
```

Comment:
@aliakseihaidukou  I'm curious why this class become abstract during a unit test fix commit.